docs/advanced-entity-analytics/entity-store.asciidoc

```diff
@@ -0,0 +1,52 @@
+[[entity-store]]
+= Entity store
+
+preview::[]
+
+.Requirements
+[sidebar]
+--
+To use the entity store, you must have the appropriate privileges. For more information, refer to <<ers-requirements, Entity risk scoring requirements>>.
+--
+
+The entity store allows you to query, reconcile, maintain, and persist entity metadata from various sources such as:
+
+* Ingested log sources
+* Integrated identity providers (such as Active Directory, EntraID, and Okta)
+* Internal and external alerts
+* External asset repositories
+* Asset criticality submissions
+
+The entity store can store any entity type observed by {elastic-sec}. This allows you to store, view, and query select entities directly from an index without needing to perform real-time searches of observable data. The entity store extracts entities from all indices in the {elastic-sec} <<default-data-view-security, default data view>>.
```

Comment:
```suggestion
The entity store can include any entity type observed by {elastic-sec}. It allows you to view and query select entities represented in your indices without needing to perform real-time searches of observable data. The entity store extracts entities from all indices in the {elastic-sec} <<default-data-view-security, default data view>>.
```
A suggestion that uses the word "store" a bit less. Not sure my suggestion is accurate but I thought that might be good since "store" hasn't really been defined in this context yet.